.github/workflows/testground.yml

```diff
@@ -20,7 +21,7 @@ jobs:
           path: "code/go-nitro-testground"
           ref: main
       
-      # Set up caching for docker (which testground uses) and go modules
+      # Set up caching for go modules
```

Comment:
How important is this / what is it doing exactly?

Reply:
The docker layer caching action was providing caching when we triggered `docker build` (by calling `make install` to build testground). Now that we're not manually triggering `docker build` I don't think the action provides any caching for us.

Reply:
The comment implies there is also some caching for `go modules`? How about that?

Reply:
https://github.com/statechannels/go-nitro/pull/903/commits/11ebb7246f7c2f768662d0fb0ad8bc79a57c4e54